backend/spec/graphql/types/query_type_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe Types::QueryType do
+  describe 'journals' do
+    before do
+      create_list(:journal, 3)
+    end
+
+    let(:query) do
+      <<~QUERY
+        query GetAllJournals {
+          journals {
+            id
+            title
+            content
+            userId
+          }
+        }
+      QUERY
+    end
+
+    let(:result) do
+      BackendSchema.execute(query)
+    end
+
+    it '全てのjournalを取得できる' do
+      expect(result.dig('data', 'journals').size).to eq(Journal.all.count)
```

Comment:
nits

```suggestion
      expect(result.dig('data', 'journals').size).to eq(Journal.count)
```

Reply:
imo

カウントの数だけでなく、取得したデータの値が期待するものかまでテストできたらベストだと思いました！

```suggestion
      expect(result).to have_attributes(
        data: {
          journals: [
            ...,
            ...,
            ...
          ]
        }
      )
```

Reply:
なんか無意識にresolvedにしてしまっていた？上のsuggestion取り込んだから自動で閉じたかな？
もう１回開きます

Reply:
@mpg-shu-shikata @yuki-snow1823 
しかたさんの提案、ちょっと反映してみたのでレビュー願う〜〜

Reply:
テスト環境のDBを参照していると思ったら、テストで開発環境のDBを使用しているんですね！
このPRのスコープ外なので別途調査したいですね〜🤔


```ruby
Failures:

  1) Types::QueryType journals 全てのjournalを取得できる
     Failure/Error: expect(result.dig('data', 'journals').size).to eq(3)

       expected: 3
            got: 68

       (compared using ==)
     # ./spec/graphql/types/query_type_spec.rb:27:in `block (3 levels) in <top (required)>'

Finished in 0.79066 seconds (files took 3 seconds to load)
1 example, 1 failure
```

Reply:
🤔
```ruby
irb(#<RSpec::ExampleGroups::TypesQueryType::Journals:0x0000ffffa982e2a0>):002:0> Rails.env.test?
=> false
irb(#<RSpec::ExampleGroups::TypesQueryType::Journals:0x0000ffffa982e2a0>):003:0> Rails.env.development?
=> true
```